### DISCUSSION TITLE
Make recorded URL easier to find

### DISCUSSION THREAD

**Discussion by ceceliacreates** [RecordReplay/devtools#7086] [General]
**Description:**

Currently, you have to hover over the root URL in the left sidebar to see locations being recorded during a replay, or log window.location. For debugging issues where the URL is important information, it would be helpful to have an option to view the current URL on screen or make this information more accessible.

**Steps to Reproduce:**

**Expected Result:**

**Additional Context:**

* Screenshots:
* Video (Loom):
* Replay ([replay.io/record-bugs](https://replay.io/record-bugs) for instructions):

https://app.replay.io/recording/replay-hover-on-urls--4a6a8382-8044-4ae8-bc52-967994a610ba?point=87620009549827587972940666291355648&time=41282&hasFrames=false

**Comment by gideonred** [comment]
I was just looking for this in the ticket queue because I was sure we've had tickets before about this.

But +1. I've probably typed window.location.href into a Replay about 30 times in the last 2 business days.

I didn't know about the hovering, so that's nice. But it doesn't help because I need to be able to copy and paste the URL (e.g. recordingIds in Replay URLs).

**Comment by jasonLaster** [comment]
Moving to a discussions so we can get more thoughts from the community as well
